src/components/CourseModal/index.tsx

```diff
@@ -238,17 +249,22 @@ export default function CourseModal({
                 mt: 'auto',
                 pt: 3,
                 display: 'flex',
-                flexDirection: { xs: 'column-reverse', sm: 'row' },
+                justifyContent: 'space-between',
+                flexDirection: { xs: 'coumn', sm: 'row' },
```

Comment:
```suggestion
                flexDirection: { xs: 'row', sm: 'row' },
```
With this layout row would be better